pages/installation/docker.rst

```diff
@@ -79,6 +79,8 @@ Example::
       image: docker.elastic.co/elasticsearch/elasticsearch:5.5.1
       environment:
         - http.host=0.0.0.0
+        - transport.host=localhost
+        - network.host=0.0.0.0
```

Comment:
Without this i've always got "Couldn't read cluster health for indices [graylog_*] (Could not connect to elasticsearch)"